laboratory-servlet-api-integration/src/it/scala/com/wixpress/petri/test/LaboratoryPropertiesIT.scala

```diff
@@ -0,0 +1,23 @@
+package com.wixpress.petri.test
+
```

Comment:
It's in there since the file that gets changed is on that classpath, we can easily move that and add a test resource version of that file it it's prefered